_health-care/_js/components/financial-assessment/Child.jsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+
+import ErrorableCheckbox from '../form-elements/ErrorableCheckbox';
+import ErrorableSelect from '../form-elements/ErrorableSelect';
+import ErrorableTextInput from '../form-elements/ErrorableTextInput';
+import DateInput from '../form-elements/DateInput';
+import FullName from '../questions/FullName';
+import SocialSecurityNumber from '../questions/SocialSecurityNumber';
+
+import { childRelationships } from '../../utils/options-for-select.js';
+import { isBlank, isValidMonetaryValue } from '../../utils/validations';
+
+// TODO: create unique nodes for each child in applicationData
+
+class Child extends React.Component {
+  render() {
+    const message = 'Please enter only numbers and a decimal point if necessary (no commas or currency signs)';
+
+    return (
+      <div>
+        <div className="row">
+          <div className="small-12 columns">
+            <p>Child's Name</p>
+            <FullName
+                value={this.props.data.childFullName}
+                onUserInput={(update) => {this.props.onStateChange('childFullName', update);}}/>
+          </div>
+        </div>
+
+        <div className="row">
+          <div className="small-12 columns">
+            <p>Child's relationship to you (check one)</p>
```

Comment:
I don't think we need this `<p>` tag, since we have a label on `ErrorableSelect`.